src/Resources/LiveTests/msgraphtest2.cer

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Does it have security concern when exposing the certificate? Is it possible to generate it by the command New-SelfSignedCertificate?

Reply:
this is a random certificate existed in our repo for quite a while:
https://github.com/Azure/azure-powershell/blob/main/src/Resources/MSGraph.Autorest/test/msgraphtest2.cer